services/comm/spec/models/template_spec.rb

```diff
@@ -5,8 +5,44 @@
 RSpec.describe Template, type: :model do
   include_examples 'application record concern' do
     let(:tenant) { Tenant.first }
-    let!(:subject) { create(factory_name) }
+    let!(:first_name) { 'Jim' }
```

Comment:
This does not need to live here. We're including the examples 'application record concern' which are shared examples. None of the tests in that shared context need any of these records to be created.

I believe you might need some of them for the render context, so then these should be created in the render context and not somewhere else randomly.